src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -35,7 +35,20 @@ object AutoExcuse : Module() {
             "Wow, so many try hards",
             "Lagggg",
             "I wasn't trying",
-            "I'm not using $CLIENT_NAME client"
+            "I'm not using $CLIENT_NAME client",
+            "Thers to much lag",
+            "My dog ate my pc",
+            "Sorry, %CLIENT_NAME Client is really bad",
```

Comment:
`%` is not detected here, use `$`

Reply:
Ye sorry, idk why I just press 5 instead of 4.